Userland/Libraries/LibWeb/HTML/HTMLCanvasElement.idl

```diff
@@ -16,8 +16,8 @@ interface HTMLCanvasElement : HTMLElement {
 
     RenderingContext? getContext(DOMString contextId, optional any options = null);
 
-    USVString toDataURL(optional DOMString type = "image/png", optional double quality);
-    undefined toBlob(BlobCallback _callback, optional DOMString type = "image/png", optional double quality);
+    USVString toDataURL(optional DOMString type = "image/png", any... arguments);
+    undefined toBlob(BlobCallback _callback, optional DOMString type = "image/png", any... arguments);
```

Comment:
Why are these `any...`? According to the spec (https://html.spec.whatwg.org/multipage/semantics.html#htmlcanvaselement) these should be `optional any quality`.

Reply:
Oops. I googled and first thing Google gave me was [this](https://www.w3.org/TR/2021/NOTE-html53-20210128/semantics-scripting.html#ref-for-dom-htmlcanvaselement-todataurl).